Makefile

```diff
@@ -297,7 +297,7 @@ verify-go-mod: update-go-mod ## Verify the go modules
 
 .PHONY: verify-deployments
 verify-deployments: deployments ## Verify the generated deployments
-	hack/tree-status
+	hack/tree-status || hack/tree-status || hack/tree-status
```

Comment:
I'd rather remove the `verify-deployments` from the verify target for now.

Reply:
There is still some value in running this as it indicates us if a contributor has run `make deployments`

Reply:
But then we have to run `make deployments` as well, not only `hack/tree-status`, right?

Reply:
duh... you're right. I got confused